src/components/calcite-button/calcite-button.tsx

```diff
@@ -169,20 +170,33 @@ export class CalciteButton {
 
   //--------------------------------------------------------------------------
   //
-  //  Private State/Props
+  //  Public Methods
   //
   //--------------------------------------------------------------------------
 
-  /** @internal */
-  /** if button type is present, assign as prop */
-  private type?: string;
+  @Method()
+  async setFocus() {
+    setElementFocus(this.childEl);
```

Comment:
I think you can just call `this.childEl.focus()` here. `setElementFocus` will be more for a parent component to focus a child component that it doesn't know if it can use setFocus or just focus

Reply:
Yep... should I just remove that `setElementFocus` helper for now? If I leave it in the utils file as `setElementFocus(el: HTMLElement)`, it I get a `Property 'setFocus' does not exist on type 'never'`.

Reply:
Yeah, just remove it for now. In the future it will help if we need to be able to set focus to one of our components